src/Avalonia.Visuals/Media/GeometryDrawing.cs

```diff
@@ -37,7 +37,8 @@ public override void Draw(DrawingContext context)
         public override Rect GetBounds()
         {
             // adding the Pen's stroke thickness here could yield wrong results due to transforms
-            return Geometry?.GetRenderBounds(0) ?? new Rect();
+            var pen = new Pen(Brushes.Black, 0);
```

Comment:
Maybe add a static readonly field with that pen?

Reply:
Yeah that might be a good idea. No idea what you'd call it though, any suggestions?

Reply:
Maybe an internal static property `Pen.ZeroThicknessPen`?

Reply:
Well that pen only seems to be used in that place, and actually, i'm not sure that comment is still true now that we have `ITransformedGeometry`; at least on D2D... So not sure that's necessary.